src/openhmd.c

```diff
@@ -491,6 +515,13 @@ void ohmd_set_default_device_properties(ohmd_device_properties* props)
 	props->ipd = 0.061f;
 	props->znear = 0.1f;
 	props->zfar = 1000.0f;
+	props->universal_distortion_k[0] = 0.0;
```

Comment:
I wasn't sure what the right thing to do when the device is uncalibrated.  The coefficients in there do no distortion correction.  Assuming DK2 lenses is probably okay too.  Is that really preferable?